mautic/mautic
# [UX] Sections for user management

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

Adds sections when editing users and fixes the buggy mobile layout.

https://github.com/mautic/mautic/pull/13473#issuecomment-1968182405



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Click to open the right sidebar
3. Go to Users
4. Click to open user details
5. See those new sections

Before:
![image](https://github.com/mautic/mautic/assets/116097999/1fec2ae8-489a-4b54-adeb-882e1ae86b24)

![image](https://github.com/mautic/mautic/assets/116097999/2f775c81-7cae-4ab1-a086-c825e231c204)

After:
![image](https://github.com/mautic/mautic/assets/116097999/477c8641-bdcc-4633-a72e-0c46de86592f)

![image](https://github.com/mautic/mautic/assets/116097999/ae7212dc-34f8-47e3-9482-c5461c69b315)
